src/codeReview.md

```diff
@@ -1,117 +0,0 @@
-% Community Code Review
```

Comment:
The contents of this file was never integrated into the Guide so I don't think it should be removed quite yet.

Reply:
I have a local copy of that content that I'll be integrating shortly so removing the file now is fine. I did however notice that the URL cr.openjdk.java.net is redirected to this file, so before we push this change to the web server we need to change that to point into the guide.